tests/unit/plugins/v1/test_nodejs.py

```diff
@@ -477,13 +480,16 @@ class Options:
         nodejs_version = nodejs._NODEJS_VERSION
         nodejs_package_manager = request.param
         nodejs_yarn_version = ""
-        source = "."
+        source_subdir = "."
```

Comment:
Then we could possibly leave this definition out (unless something using this fixture depends on the source location).